Integrations/integration-ATD.yml

```diff
@@ -37,672 +38,670 @@ configuration:
+        skipTaskId = cast_to_int(args['skipTaskId']) if 'skipTaskId' in args else None
+        xMode = cast_to_int(args['xMode']) if 'xMode' in args else None
+        messageId = args['messageId'] if 'messageId' in args else None
+        filePriorityQ = args['filePriorityQ'] if 'filePriorityQ' in args else None
+        srcIp = args['srcIp'] if 'srcIp' in args else None
+        destIp = args['destIp'] if 'destIp' in args else None
+        fileName = args['fileName'] if 'fileName' in args else None
+
+        result = file_upload(cast_to_int(args['submitType']), sample, vmProfileList,
+                            skipTaskId, analyzeAgain, xMode, messageId, filePriorityQ,
+                            srcIp, destIp, fileName)
+        md = tableToMarkdown('ATD sandbox sample submission', prettify_file_upload_res(result['resultObj']),
+                                ['taskId', 'jobId', 'messageId', 'url', 'destIp', 'srcIp', 'MD5', 'SHA1', 'SHA256'])
+
+        upload_file_output = {
+            "ATD.Task(val.taskId == obj.taskId)": prettify_file_upload_res(result['resultObj']),
+            "ATD.taskId": result['taskId'] # backward compatabillity
         }
-        return checkTaskStatusByTaskId(taskIds);
-    }
-
-    function checkTaskStatusCommand(){
-        var ids = [];
-        var result;
+        if 'url' in args:
+            upload_file_output["URL(val.Data == obj.Data).Data"] = sample
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': result['resultObj'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': upload_file_output
+        })
+
+    def build_report_context(reportSummary, uploadData, status, threshold):
+        context = {}
+        if ((reportSummary) and (reportSummary['Subject'])):
+            subject = reportSummary['Subject']
+            context = { 'DBotScore': {'Vendor': 'McAfee-Advanced-Threat-Defense', 'Score': 0 }}
+            if ('FileType' in subject):
+                context['DBotScore']['Indicator'] = subject['md5']
+                context['DBotScore']['Type'] = 'hash'
+                # default threshold for McAfee ATD is 3
+                if (reportSummary['Verdict']['Severity'] > threshold):
+                    context['DBotScore']['Score'] = 3
+                    if subject['Type'] == 'application/url':
+                        context['URL(val.Name == obj.Data)'] = {
+                            'Type' : subject['Type'],
+                            'MD5' : subject['md5'],
+                            'SHA1' : subject['sha-1'],
+                            'SHA256' : subject['sha-256'],
+                            'Size' : subject['size'],
+                            'Name' : subject['Name'],
+                            'Malicious': {
+                                'Vendor': 'McAfee-Advanced-Threat-Defense',
```

Comment:
What are the dashes for?

Reply:
best to write just with spaces i guess. will do